src/amdilc/amdilc_dump.c

```diff
@@ -498,6 +507,9 @@ static void dumpInstruction(
     case IL_OP_BREAK:
         fprintf(file, "break");
         break;
+    case IL_OP_BREAKC:
+        fprintf(file, "breakc_relop(%s)", GET_BITS(instr->control, 0, 2) >= 6 ? "unknown" : mIlRelopNames[GET_BITS(instr->control, 0, 2)]);
```

Comment:
drop the range check and it's good to go. we only dump shaders for debugging anyway.